exporter/trace/trace_proto.go

```diff
@@ -71,11 +71,29 @@ func generateDisplayName(s *export.SpanData, format DisplayNameFormatter) string
 	}
 }
 
+func injectLabelsFromResources(sd *export.SpanData) {
+	if sd.Resource.Len() == 0 {
+		return
+	}
+	uniqueAttrs := make(map[kv.KeyValue]bool)
+	for _, attr := range sd.Attributes {
+		uniqueAttrs[attr] = true	
+	}
+	for _, attr := range sd.Resource.Attributes() {
+		if !uniqueAttrs[attr] {
```

Comment:
It's more readable to use indentation for non-happy path. See article https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88

So I suggest to rewrite this block as follows:
```go
if uniqueAttrs[attr] {
  continue // skip resource attributes which conflict with span attributes
}

uniqueAttrs[attr] = true
sd.Attibutes = append(sd.Attributes, attr)
```